server/service/src/sync/translations/invoice_line.rs

```diff
@@ -95,6 +95,9 @@ pub struct LegacyTransLineRow {
     #[serde(deserialize_with = "empty_str_as_option_string")]
     #[serde(rename = "vaccine_vial_monitor_status_ID")]
     pub vvm_status_id: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    pub campaign_id: Option<String>,
```

Comment:
I am thinking that we don't need legacy translation here because campaign table  isn't in OG (only in OMS).

However I'm not super confident with this part of the codebase. @jmbrunskill, would you be willing to confirm if we need this?

Reply:
Yes, we should add to both invoice_line and stock_line in OMS. As discussed lets do it as an `OMS_fields` where we can extend the JSON in OMS without having to do an mSupply version change again in future.

Reply:
Thanks @jmbrunskill, have created a separate issue for it: https://github.com/msupply-foundation/open-msupply/issues/7882